src/System.Windows.Forms/src/System/Windows/Forms/SaveFileDialog.cs

```diff
@@ -22,6 +23,9 @@ namespace System.Windows.Forms
     ]
     public sealed partial class SaveFileDialog : FileDialog
     {
+        private static Guid IID_IFileSaveDialog = new Guid("84bccd23-5fde-4cdb-aea4-af64b83d78ab");
+        private static Guid CLSID_FileSaveDialogCoClass = new Guid("C0B4E2F3-BA21-4773-8DBA-335EC946EB8B");
```

Comment:
```suggestion
        private static Guid CLSID_FileSaveDialog = new Guid("C0B4E2F3-BA21-4773-8DBA-335EC946EB8B");
```

nit: the others are named correctly, this forgot to remove the CoClass suffix (implied by `CLSID`, usually not part of the name)